sdk/selfhelp/arm-selfhelp/src/operations/discoverySolutionNLPSubscriptionScope.ts

```diff
@@ -33,12 +33,17 @@ export class DiscoverySolutionNLPSubscriptionScopeImpl
   /**
    * Search for relevant Azure Diagnostics, Solutions and Troubleshooters using a natural language issue
    * summary and subscription.
+   * @param subscriptionId The Azure subscription ID.
    * @param options The options parameters.
    */
   post(
+    subscriptionId: string,
```

Comment:
@kazrael2119 could you verify the change?

Reply:
change from https://github.com/Azure/azure-rest-api-specs/pull/28935/files